stack-definition.yaml

```diff
@@ -1408,7 +1291,18 @@ components:
         }
       ]
 changeLog: |
-  - 🍎 The HelmRelease kind was promoted from v2beta1 to v2beta2.
+  - 🦋 Upgraded all stack components to latest version possible!
+  - 🍏 External Secrets upgraded to v0.9.11
+  - 🍏 Grafana upgraded to 10.3.1
+  - 🍏 Upgraded onechart to 0.65.0 for Image Builder
+  - 🍏 Ingres Nginx upgraded to 1.9.6
+  - 🍏 Keda upgraded to 2.13.0
+  - 🍏 Prometheus Node Exporter upgraded to 1.7.0
+  - 🍏 Prometheus upgraded to v2.49.1
+  - 🍏 Promtail upgraded to 2.9.3
+  - 🍏 Sealed Secrets upgraded to 0.25.0 v1.13.3
+  - 🍏 Cert Manager upgraded to v1.13.3
+
```

Comment:
We deprecated Kyverno, maybe others. Those are breaking changes and we should advise people how they can keep their existing Kyvernos. Will it be deleted if they update?